converter/converters/kube_validwhconfig_to_koki_validwhconfig.go

```diff
@@ -0,0 +1,65 @@
+package converters
+
+import (
+	admissionregv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+
+	"github.com/koki/short/types"
+)
+
+func Convert_Kube_ValidatingWebhookConfiguration_to_Koki_ValidatingWebhookConfiguration(kubeInitConfig *admissionregv1beta1.ValidatingWebhookConfiguration) (*types.ValidatingWebhookConfigWrapper, error) {
+	var err error
+	kokiWrapper := &types.ValidatingWebhookConfigWrapper{}
+	kokiInitConfig := &kokiWrapper.ValidatingWebhookConfig
+
+	kokiInitConfig.Name = kubeInitConfig.Name
+	kokiInitConfig.Namespace = kubeInitConfig.Namespace
+	kokiInitConfig.Version = kubeInitConfig.APIVersion
+	kokiInitConfig.Cluster = kubeInitConfig.ClusterName
+	kokiInitConfig.Labels = kubeInitConfig.Labels
+	kokiInitConfig.Annotations = kubeInitConfig.Annotations
+
+	rules, err := convertValidatingWebhooks(kubeInitConfig.Webhooks)
+	if err != nil {
+		return nil, err
+	}
+	kokiInitConfig.Rules = rules
+
+	return kokiWrapper, nil
+}
+
+func convertValidatingWebhooks(webhooks []admissionregv1beta1.Webhook) (map[string][]types.ValidatingWebhookRuleWithOperations, error) {
```

Comment:
check for nil webhooks - even though the unmarshalling code cannot return a nil webhook, some people (Blackduck) use the API directly, and they might pass it.